packages/vetrina/src/Vetrina/Types.purs

```diff
@@ -12,7 +12,10 @@ import React.Basic (JSX)
 
 data AccountStatus
   = NewAccount
-  | ExistingAccount String
+  | ExistingAccount
+    { email :: String
+    , invalidPassword :: Boolean
```

Comment:
Could we have a new constructor for the failed login case? To me, it would make more sense.

Reply:
@kaol What do you think?